### PR TITLE
Replaces DMItool with contemporary /tg/station version.

### PR DESCRIPTION
**Pull request by alsoandanswer**
title

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: .gitattributes [hunk 1]
```diff
@@ -1,10 +1,6 @@
-# dmm map merger hook
-# needs additional setup, see tools/mapmerge/install.txt
-*.dmm merge=merge-dmm eol=crlf
-
-# dmi icon merger hook
-# needs additional setup, see tools/dmitool/merging.txt
-*.dmi merge=merge-dmi
+# merger hooks, run tools/hooks/install.bat or install.sh to set up
+*.dmm merge=dmm
+*.dmi merge=dmi
```
Comment (NonQueueingMatt) [inline .gitattributes:3]: This does not look like something we want to change. Specifically, the eol part.

**Review by NonQueueingMatt — DISMISSED** [review]

**Review by Amunak — APPROVED** [review]

File: .gitattributes [hunk 2]
```diff
@@ -13,7 +9,7 @@ html/changelog.html merge=union
 * text=auto
 
 *.ai binary
-*.dmi binary
+#*.dmi binary
```
Comment (Arrow768) [inline .gitattributes:12]: delete that instead of commenting it out
```suggestion
```

File: .gitattributes [hunk 3]
```diff
@@ -40,7 +36,7 @@ html/changelog.html merge=union
 *.css text
 *.dm text
 *.dmf text
-*.dmm text
+#*.dmm text
```
Comment (Arrow768) [inline .gitattributes:39]: delete that instead of commenting it out
```suggestion
```

File: .gitattributes [hunk 4]
```diff
@@ -1,10 +1,6 @@
-# dmm map merger hook
-# needs additional setup, see tools/mapmerge/install.txt
-*.dmm merge=merge-dmm eol=crlf
-
-# dmi icon merger hook
-# needs additional setup, see tools/dmitool/merging.txt
-*.dmi merge=merge-dmi
+# merger hooks, run tools/hooks/install.bat or install.sh to set up
+*.dmm text merge=dmm
```
Comment (Arrow768) [inline .gitattributes:2]:
```suggestion
*.dmm text merge=dmm eol=crlf
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**